### ISSUE TITLE
Game crashes when select jus chars

### ISSUE THREAD

**Issue by YanisCr777** [qingjun1991/GameDreamFactory#14]
Game crashes when select jus chars

**Comment by qingjun1991** [comment]
1. Maybe you haven't updated the latest patch
2. Maybe you added too many characters

**Comment by YanisCr777** [comment]
Bro I downloaded the latest patch but I don't how to set up the latest one please help me I am a big fan of you and I will support your work always as much as I can ❤

**Comment by YanisCr777** [comment]
Bro you mentioned too many chars if I add only 6chars but they are all big characters like more than 100mb
can I fix it ?

**Comment by qingjun1991** [comment]
Simply overwrite the folder and the update will be successful.

**Comment by YanisCr777** [comment]
bro I don't know how to set latest patch can you help me ok I will tell you
what I did I extract the (GameDreamFactory.2024-03-30.patch) file to my
storage and installed the apk then copied my mugen to games then oppened
apk and I select game dream factory folder and nothing is happening 🥺

**Comment by qingjun1991** [comment]
You just need to make sure that there is a prompt when overwriting the folder whether to overwrite the file with the same name. If there is no prompt, it means the path is incorrect, otherwise it is correct

After agreeing to overwrite the file with the same name, the set latest patch is successful.

**Comment by qingjun1991** [comment]
"\!GameDreamFactory\log.txt"
You open this file and see what error is reported

**Comment by YanisCr777** [comment]
Still crashes when select heavy characters more than (300MB) but it's Ok
thanks anyway bro I appreciate your work and I like it I hope you will
upload it on youtube soon people absolutely will love it. waiting for your
next update ❤

**Comment by qingjun1991** [comment]
Six characters are not too many, and 300mb is not too large, but the character compression file must be decompressed to ensure that there is no compression file in the chars folder.

**Comment by YanisCr777** [comment]
No, Bro I meant (6 chars x 300mb) each one has more than 300mb raw folder,
I think it is just too heavy because they are full HD chars and has HD
effects and spirites including the screenpack HD quality too and has
dynamic intros so the game dream factory folder size is now arround 3gb 😕

**Comment by qingjun1991** [comment]
Oh, it's so crazy!

**Comment by YanisCr777** [comment]
Thanks Bro 😂

**Comment by qingjun1991** [comment]
In my country, I am unable to access YouTube.

You mentioned earlier about "select game dream factory folder".
Do you mean that you didn't put "!GameDreamFactory" correctly placed in "/storage/emulated/0/!GameDreamFactory", which requires selecting a folder? Is a bad practice as it can lead to the app taking up more memory, making it easier to crash, and loading slower.

**Comment by YanisCr777** [comment]
No, I added the game dream folder in external storage correctly in
storage/emulated/0/ but I don't know why the game still crashes I think it
is because of ram my phone has only 4gb ram but it works smoothly but the
emulator crashes when I select heavy characters after 10min no more
is there any solutions 😭 or my phone too old So I need to buy a new one ?

**Comment by qingjun1991** [comment]
https://github.com/qingjun1991/GameDreamFactory/issues/8
